packages/InteractiveProfilingTool-Core.package/IPTMessageTallyWrapper.class/instance/getTenures.st

```diff
@@ -0,0 +1,6 @@
+information extraction
+getTenures
+	
+	self messageTally gcStats ifNil: [^ nil].
```

Comment:
For consistency reasons please change to
```smalltalk
self messageTally gcStats 
    ifNil: [^ nil].
```

Reply:
Ok wurde gemacht.